tensorboard/plugins/interactive_inference/WIT_from_scratch.ipynb

```diff
@@ -6,24 +6,46 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "## What-If Tool from scratch - From CSV to trained model to What-If Tool usage\n",
+    "## What-If Tool from scratch - From CSV to trained binary classification model to What-If Tool usage\n",
     "\n",
     "# This notebook shows the process of loading up a dataset from CSV, training a very simple classifier to\n",
     "# predict one of the columns, then using the What-If Tool (WIT) to analyze the training dataset and the trained\n",
     "# model.\n",
     "\n",
-    "# It is shown with both the UCI census binary classification task and the UCI iris multiclass classification task.\n",
+    "# This notebook uses the UCI Census dataset and learning problem, detailed at\n",
+    "# https://archive.ics.uci.edu/ml/datasets/census+income, which predicts whether a person earns more than $50k\n",
+    "# given their census information.\n",
+    "# To customize this notebook to work on your own dataset, you only need to edit the sections marked with \"USER: \"\n",
     "\n",
     "## Setup (install Jupyter, TF, and TF Serving in a virtualenv).\n",
     "# NOTE: Use of a virtualenv, pip installation of tensorflow and docker use for TF Serving aren't the only way\n",
-    "# to set all this up. I just find it the simplest to use.\n",
-    "\n",
-    "# virtualenv tf\n",
-    "# source tf/bin/activate\n",
-    "# pip install --upgrade pip\n",
-    "# pip install jupyter\n",
-    "# pip install tensorflow (or tensorflow-gpu)\n",
-    "# docker pull tensorflow/serving"
+    "# to set all this up. I just find it the simplest and safest to use.\n",
+    "\n",
+    "# Step 1: Install pip\n",
+    "# Mac: > sudo easy_install pip\n",
```

Comment:
I am missing a lot of context so please regard this comment mostly as a question:
on more recent macOS, using sudo is discouraged. Also since macs often ship with python, shouldn't pip be already there? How about some linkage to here: https://www.tensorflow.org/install/pip

Reply:
updated the instructions to point to that page instead of listing out my own steps